frame/balances/src/tests.rs

```diff
@@ -692,15 +692,15 @@ macro_rules! decl_tests {
 					);
 
 					System::set_block_number(3);
-					let _ = Balances::unreserve(&1, 5);
+					assert!(Balances::unreserve(&1, 5).is_zero());
```

Comment:
I see some `let _ = ...` above.

Looks like they should be wrapped in `assert_ok!(..)`

Reply:
I can do it, it is not directly related to this PR but I can change it as I change this file.

Reply:
So I'm wrapping all Result-returning functions, but I can leave out the ones that are unfailable ones like `slash` or `deposit_creating` right?

Reply:
yup!

Reply:
ill fix it for you real quick, just a nit pick